### PR TITLE
added some addresses we collected

### PR DESCRIPTION
**Pull request by chebelom**

### CODE REVIEW

File: disposable_email_blacklist.conf [hunk 1]
```diff
@@ -157,6 +160,7 @@ centermail.com
 centermail.net
 chammy.info
 cheatmail.de
+chickenkiller.com
```
Comment (martenson) [inline disposable_email_blacklist.conf]: this does not look like a disposable email domain - http://chickenkiller.com/

File: disposable_email_blacklist.conf [hunk 2]
```diff
@@ -728,6 +741,7 @@ nomorespamemails.com
 nonspam.eu
 nonspammer.de
 noref.in
+nospam.wins.com.br
```
Comment (martenson) [inline disposable_email_blacklist.conf]: com.br is not a valid domain name

File: disposable_email_blacklist.conf (hunk 1 above)
Comment (chebelom) [inline disposable_email_blacklist.conf]: fixed, sorry (:

**Comment by martenson** [comment]
thank you very much @chebelom !

**Pull request merged**